schemas/preferences.sublime-settings.json

```diff
@@ -206,12 +217,51 @@
           "default": "auto"
         },
         "rulers": {
-          "markdownDescription": "Columns in which to display vertical rulers.",
+          "markdownDescription": "Columns in which to display vertical rulers.\n\nAs of ST 4133 column and style (see `ruler_style`) can be paired.\n\ne.g.: `\"rulers\": [[80, \"solid\"], [100, \"dotted\"]]`",
```

Comment:
I guess that's a general issue/question wheter to note new settings or changes in possible values depending onversions. LSP can be installed by ST users, which may have decided to stick with an older stable release (e.g.: 4126). Those would get the new suggestions (settings and value formats) as well and might wonder, why they don't work. So some hints may in general be a good idea. It effects all (new/changed) settings though.

Reply:
Yes, it would be better to have version information but it's extra work that someone would have to do and I don't care enough to ensure that this is always present and correct.

Many existing options don't have such information. Heck, even some added here don't have it.

Reply:
I've always tracked _Default_ package changes in a repo. So it's not too difficult to find those changes. That's acutally what I did in this PR as well. Maybe find the time to add some infos about it at some point.